<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>434</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Source Code Pro Medium</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>KLUB</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="4">
         <widget class="QPushButton" name="buttonSoupiska">
          <property name="text">
           <string>Soupiska</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="buttonOdebratKlub">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="buttonPridatKlub">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Kluby</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QTableView" name="tableViewKlub"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>SOUPISKA</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Přehle hráčů týmu</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>246</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="QPushButton" name="hrac_pridat">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" rowspan="2">
         <widget class="QPushButton" name="hrac_odebrat">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5" rowspan="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_nazevTymu">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="6">
         <widget class="QTableView" name="tableView_hrac"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Realizační tým</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="6">
         <widget class="QTableView" name="tableView_Trener"/>
        </item>
        <item row="3" column="5">
         <widget class="QPushButton" name="trener_odebrat">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="trener_pridat">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>265</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>546</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
